comparisons/comparison-bpmn.md

```diff
@@ -309,14 +309,30 @@ functions:
 id: subflowloop
 name: SubFlow Repeat workflow
 version: '1.0'
-start: SubflowRepeat
+start: StartCount
 states:
+- name: StartCount
+  type: inject
+  data:
+    counter: 0
```

Comment:
I think here you dont have to use the inject state. Just assume that "count" (I would call it "max") is passed as workflow data input.

Reply:
Done